templates/practice/documentation.html

```diff
@@ -100,20 +106,10 @@ <h4 class="p-heading--3">Equipment</h4>
-        <p>Their work shapes the way our products are conceived and presented. Amongst other things, it's reflective, critical, active and creative. It's work that feeds back into the product itself, and helps define what it is, and always seeks new and better ways to add to its value.</p>
-        <p>Our Technical Authors come from many backgrounds, including academic research, software engineering and journalism. What they have in common is that they are expert, effective communicators. They share an intense sense of technical curiosity, and a strong feeling for the perspective of a product's users. Their strength of imagination and qualities of empathy are crucial to their work.</p>
-        <p>Documentation is something that everyone contributes to; Technical Authors encourage and lead documentation work, and help guarantee its quality.</p>
         <h2>Careers and opportunities</h2>
         <p>There is a dedicated career progression in documentation at Canonical, including roles for Technical Author, Senior Technical Author and Documentation Manager.</p>
-        <p>Our plans include recruiting a cohort of technical authors to join several engineering teams, working on products across Canonical's portfolio. For those roles, you can expect to be interviewed on a wide range of matters, on the many roads that lead to documentation.</p>
-        <p>Documentation is not a siloed activity. You'll find engineers and product managers making real and substantial contributions, as part of their everyday work.</p>
-        <p>It's a responsibility that is shared by everyone who works in engineering, product or support roles. We expect everyone to help manage that responsibility.</p>
-        <p>If you're applying to Canonical for an engineering role, don't be surprised to be asked about your work in &dash; or thoughts on &dash; documentation at a job interview. As an engineering manager or director, expect to sit an entire interview dedicated to documentation practice.</p>
+        <p>We’re hiring technical authors to join several engineering teams, working on products across Canonical's portfolio - see <a href="https://docs.google.com/document/d/1U-IkZOMzQlfV0zAAmM0PRVUBoZNBeOa-Qrso3i4rDVU/edit?usp=sharing">Working in documentation at Canonical</a> for more.</p>
+        <p>And if you’re applying to Canonical for an engineering role , don’t be surprised to be asked about your work in - or thoughts on - documentation at a job interview. As an engineering manager or director, expect to sit an entire interview dedicated to documentation practice. </p>
         <div class="p-notification--information">
           <div class="p-notification__content">
             <p class="p-notification__message">Whatever the role, don't hesitate to mention documentation-related work that you're proud of in your application or your interview &dash; it will be valued.</p>
```

Comment:
```suggestion
            <p class="p-notification__message">Whatever the role, don't hesitate to mention documentation-related work that you're proud of in your application or your interview &mdash; it will be valued.</p>
```

Reply:
Thanks, updated.